assignments/arrays.js

```diff
@@ -63,32 +63,50 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(inventory[33].car_year+inventory[33].car_make+inventory[33].car_model);
 
```

Comment:
Remember that indexes in arrays start at 0 so the car with the id of 33 would actually be `inventory[32]`
You can set it as variable `const car33 = inventory[32];` then console log  it with string interpolation so its easier to find in the console. ```console.log(`Car 33 is a ${car33.car_year} ${car33.car_make} ${car33.car_model}`);```
```